site_root/input/resources/examples/Task-example-research-study-task-id.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Task",
+  "id": "example-research-study-task-id",
+  "status": "accepted",
+  "intent": "unknown",
```

Comment:
Per http://hl7.org/fhir/R4/valueset-task-intent.html, `"order"` since the sequencing task is ordered by the PI?

Reply:
resolved - implemented.